src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingViewExtensions.android.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+using CommunityToolkit.Maui.Core.Views;
+using Microsoft.Maui.Platform;
+
+namespace CommunityToolkit.Maui.Core.Extensions;
+
+/// <summary>
+/// Extension methods to support <see cref="IDrawingView"/>
+/// </summary>
+public static partial class MauiDrawingViewExtensions
+{
+	/// <summary>
+	/// Set MultiLine mode
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="multiLineMode">value</param>
+	public static void SetMultiLineMode(this MauiDrawingView mauiDrawingView, bool multiLineMode)
+	{
+		mauiDrawingView.MultiLineMode = multiLineMode;
+	}
+
+	/// <summary>
+	/// Set ClearOnFinish
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="clearOnFinish">value</param>
+	public static void SetClearOnFinish(this MauiDrawingView mauiDrawingView, bool clearOnFinish)
+	{
+		mauiDrawingView.ClearOnFinish = clearOnFinish;
+	}
+
+	/// <summary>
+	/// Set LineColor
+	/// </summary>
+	/// <param name="mauiDrawingView"><see cref="MauiDrawingView"/></param>
+	/// <param name="lineColor">line color</param>
+	public static void SetLineColor(this MauiDrawingView mauiDrawingView, Color lineColor)
+	{
+		mauiDrawingView.LineColor = lineColor.ToPlatform();
```

Comment:
this should be in an extension method

Reply:
But this is already an extension method. Am I missed something?

Reply:
Everything after the = should be in an extension method.
That's needed because if someone want to override this behavior and call the original implementation, they will be able to do.

Reply:
Sorry, still don't understand. I created SetLineColor extension method to set line color. What another extension method am I need?

Reply:
Oh boy, you're right... I just messed up with the files. Sorry for that